setup.py

```diff
@@ -0,0 +1,45 @@
+import setuptools
+import sys
+
+tests_require = [line.strip() for line in open('test-requirements.txt', 'r')]
+if sys.version_info < (2, 7):
+    tests_require.append('unittest2')
+if sys.version_info < (3, 0):
+    tests_require.append('mock')
+
+setuptools.setup(
+    name='sprockets.handlers.status',
+    version='0.0.0',
```

Comment:
While it wouldn't cause issues with this package, it can when you include dependencies that have yet to be installed at the time of ``python setup.py install``. I try and avoid that practice in general these days.